lib/employees/resolvers.ts

```diff
@@ -149,7 +149,8 @@ export const updateEmployee: Resolver = async (_, args, { prisma }) => {
  * @returns status of operation (ok)
  */
 export const deleteEmployee: Resolver = async (_, args, { prisma }) => {
-  const id = parseInt(args.id);
+  const id = parseInt(args.input.id);
+
```

Comment:
Sorry this wasn't included in the original ticket, but for the update and create endpoints, we're returning the updated/created employee record from the API so that we can grab the name of the employee that was affected. We want to do the same for this endpoint, ie. return the deleted employee, to enforce data integrity (getting the data from the DB instead of the frontend). Please see https://github.com/uwblueprint/richmond-centre-for-disability/pull/85 for an example